### PR TITLE
Use passed jQuery (jquery) instance rather than $

### PR DESCRIPTION
**Pull request by 4ver**
This is to fix cases where someone has used `jQuery.noConflict()`.

grunt-umd isn't flexible enough to define the name of the variable it gives you for jQuery so I just replaced all instances of `$` with `jquery` (the passed jQuery instance).

Cheers,
Donal.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[*]
+charset = "utf-8"
+
+[*.coffee]
+indent_style = space
+indent_size = 2
+trim_trailing_whitespace = true
```
Comment (ichord) [inline .editorconfig]: Don't add it.

**Comment by ichord** [comment]
Why don't just assign `jquery` to `$` at the beginning?

```javascript
}(this, function (jquery) {

var $ = jquery;

var Api, App, Controller, DEFAULT_CALLBACKS, EditableController, KEY_CODE, Model, TextareaController, View,
```

**Comment by XrXr** [comment]
Or name the argument `$`

```js
(function ($){
  // ...
})(jQuery)
```

File: .editorconfig (hunk 1 above)
Comment (4ver) [inline .editorconfig]: Okay- But having it will make it a bit easier for people to automatically adhere to your code style guidelines.

**Comment by 4ver** [comment]
Aha I only just saw that you specify the files to concatenate and in what order in the Gruntfile- That should work fine!

Thanks.

**Comment by 4ver** [comment]
This is sorted now- spent most of my time fixing up inconsistent indentation in the specs folder :stuck_out_tongue:

**Comment by ichord** [comment]
Great. :+1:

**Pull request merged**